app/src/main/java/io/github/gmathi/novellibrary/database/DownloadHelper.kt

```diff
@@ -20,6 +20,7 @@ fun DBHelper.createDownload(download: Download): Boolean {
     values.put(DBKeys.KEY_NAME, download.novelName)
     values.put(DBKeys.KEY_WEB_PAGE_ID, download.webPageId)
     values.put(DBKeys.KEY_CHAPTER, download.chapter)
+    // TODO: By default, downloads should be queued
```

Comment:
That was causing problems, so I had to do this. Downloads still need work, I actually need the chapter downloads, anyway I can create that once I get time to work on it.